internal/compiler/generator/cpp.rs

```diff
@@ -1831,6 +1839,7 @@ fn generate_sub_component(
                     "}} else if (index >= {} && index < {}) {{",
                     sub.index_of_first_child_in_tree,
                     sub.index_of_first_child_in_tree + sub_items_count - 1
+                        + sub.ty.repeater_count()
```

Comment:
Oops, how come this didn't bite us before?

Reply:
we don't use accessibility very often.